service/apiService.ts

```diff
@@ -0,0 +1,42 @@
+export const fetchData = async (
+  queryParam: string | undefined,
+  page: string,
+  pageSize: string
+) => {
+  let url;
+  if (queryParam) {
+    url = `https://belka.romakhin.ru/api/v1/rimorti?search.name=${queryParam}&page=${
+      Number(page) - 1
+    }&page_size=${pageSize}`;
+  } else {
+    url = `https://belka.romakhin.ru/api/v1/rimorti?page=${
+      Number(page) - 1
+    }&page_size=${pageSize}`;
+  }
+
+  try {
+    const response = await fetch(url);
+    if (!response.ok) {
```

Comment:
In try/catch block if any code in "try" is failed it immediately goes to "catch" section. It means than the next code (after failed) will be not executed. In another words if ```await fetch(url);``` fails, this "if" block will be not executed, so it redundant here.

Reply:
fixed